src/main/java/dukeclasses/Storage.java

```diff
@@ -0,0 +1,150 @@
+package dukeclasses;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Modifies the storage file that is used in permenant storage of data.
+ */
+public class Storage {
+    private String filePath;
+
+    /**
+     * Constructor for Storage.
+     *
+     * @param filePath that indicates the path of the storage file.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Returns the path of the storage file
+     *
+     * @return String representing the path of the storage file.
+     */
+    public String getStorageFilePath() {
+        return filePath;
+    }
+
+    /**
+     * Loads data of the storage file into an ArrayList</Task>. Usually called at the start of Duke.
```

Comment:
This might violate the coding standard. It mentioned 
> Write the first sentence as a short summary of the method, as Javadoc automatically places it in the method summary table (and index).

Perhaps the second sentence can be underneath the first